skills/substrate/references/subprocess-streams.md

```
@@ -0,0 +1,31 @@
+**Load this when a script launches another program.** A session that only reads or writes files does not need it.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)